apps/metadata-utils/src/types.ts

```diff
@@ -7,7 +7,9 @@ export interface ISetting {
   value: string;
 }
 
-export type ColumnType =
+export type HeadingType = "HEADING";
+
+export type ValueType =
```

Comment:
These are exported but not used outside this file. Better to not export in that case, in order to prevent dead code.